### ISSUE TITLE
Plunkers not working for md-data-table examples

### ISSUE THREAD

**Issue by gseabrook** [angular/material2#5569]
The plunker examples for all the md-data-tables are not working, I receive the following error

(SystemJS) XHR error (404) loading https://run.plnkr.co/sdILEjRNoLe9paJ1/@angular/cdk.ts

Try following any of the plunker links on

https://material.angular.io/components/table/overview

**Comment by Hypopheralcus** [comment]
Adding 

`'@angular/cdk': 'https://unpkg.com/@angular/cdk/bundles/cdk.umd.js',`

to the **Systemjs.config.js** solves the 404. 

But there is another error. After adding the wrong reference:

> Unhandled Promise rejection: Template parse errors:
> Can't bind to 'cdkHeaderRowDef' since it isn't a known property of 'md-header-row'.
> 1. If 'md-header-row' is an Angular component and it has 'cdkHeaderRowDef' input, then verify that it is part of this module.
> 2. If 'md-header-row' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.
> 3. To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component. ("
>     </ng-container>
> 
>     <md-header-row [ERROR ->]*cdkHeaderRowDef="displayedColumns"></md-header-row>
>     <md-row *cdkRowDef="let row; columns: displ"): ng:///table-basic-example.html@32:19
> Property binding cdkHeaderRowDef not used by any directive on an embedded template. Make sure that the property name is spelled correctly and all directives are listed in the "@NgModule.declarations". ("
>     </ng-container>
> 
>     [ERROR ->]<md-header-row *cdkHeaderRowDef="displayedColumns"></md-header-row>
>     <md-row *cdkRowDef="let row;"): ng:///table-basic-example.html@32:4

**Comment by Hypopheralcus** [comment]
Adding
`import { CdkTableModule } from '@angular/cdk';` to **main.ts**
and add **CdkTableModule** to the imports solves the problem.

```
imports: [
    BrowserModule,
    BrowserAnimationsModule,
    FormsModule,
    HttpModule,
    CdkTableModule,
    MaterialModule,
    MdNativeDateModule,
    ReactiveFormsModule,
  ],
```

**Comment by Hypopheralcus** [comment]
Here is a working plunker version https://plnkr.co/edit/nASMWAJxt3epEwptV4mp?

**Comment by ScottRFrost** [comment]
Wow, the examples are really hard to follow.  Can you create a plunker that shows retrieving data from a JSON API with paging?  I'd imagine that's a much more common use case than reading all of the possible data when the page loads as these examples seem to do.

**Comment by ryanpm40** [comment]
This seems to be affecting multiple plunkr examples.

_At least_ md-select and the md-button example from the overview tab. Not sure how many others.

**Comment by ryanpm40** [comment]
@Hypopheralcus Your 'working' example doesn't load for me, it's getting that "Error loading https://unpkg.com/@angular/platform-browser/bundles/platform-browser.umd.js as "@angular/platform-browser" from https://run.plnkr.co/BT3WAmHpCIXT4GAK/main.ts" error.

**Comment by Hypopheralcus** [comment]
@ryanpm40 Your plunker link doesn't work. My version is still working.

**Comment by ryanpm40** [comment]
@Hypopheralcus  The error I provided was from your plunkr link (https://plnkr.co/edit/nASMWAJxt3epEwptV4mp), I clicked the one you posted, and that's the error I received.

However, it seems to have resolved now. Maybe there was some strange downtime with https://unpkg.com at the time, thanks!

**Comment by jelbourn** [comment]
They should all be fixed now.

**Comment by ryanpm40** [comment]
@jelbourn 

They are still not working. Please see the plunker linked on the Overview tab of DatePicker. Errors are:

`Failed to load resource: the server responded with a status of 404 (Not Found)
zone.min.js:1 Error: (SystemJS) Invalid or unexpected token
	SyntaxError: Invalid or unexpected token
	    at eval (<anonymous>)
	    at e.invoke (https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:19922)
	    at t.run (https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:15134)
	    at https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:12167
	    at e.invokeTask (https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:20606)
	    at t.runTask (https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:15783)
	    at i (https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:10706)
	    at XMLHttpRequest.invoke (https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:21603)
	Evaluating https://unpkg.com/@angular/platform-browser/bundles/platform-browser.umd.js
	Error loading https://unpkg.com/@angular/platform-browser/bundles/platform-browser.umd.js as "@angular/platform-browser" from https://run.plnkr.co/M8n19lwO8hX8EWKh/main.ts
	    at eval (<anonymous>)
	    at e.invoke (https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:19922)
	    at t.run (https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:15134)
	    at https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:12167
	    at e.invokeTask (https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:20606)
	    at t.runTask (https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:15783)
	    at i (https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:10706)
	    at XMLHttpRequest.invoke (https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:21603)
	Evaluating https://unpkg.com/@angular/platform-browser/bundles/platform-browser.umd.js
	Error loading https://unpkg.com/@angular/platform-browser/bundles/platform-browser.umd.js as "@angular/platform-browser" from https://run.plnkr.co/M8n19lwO8hX8EWKh/main.ts
e.invoke @ zone.min.js:1
run.plnkr.co/M8n19lwO8hX8EWKh/@angular/cdk.ts Failed to load resource: the server responded with a status of 404 ()`

**Comment by jelbourn** [comment]
Works for me, be sure to clear your broswer cache.

**Comment by ryanpm40** [comment]
@jelbourn  Very odd. I'm seeing the issue on my own Win7 machine and a Win10 VM in chrome. I even have "Disable cache (while DevTools is open)" checked in my Chrome F12 settings. Not sure what's going on, but if it works for you I believe you, guess I'll just wait and see if it fixes itself for me.

**Comment by jelbourn** [comment]
Possibly a propagation delay on the hosting update

**Comment by harogaston** [comment]
Plunkers of the documentation don't work. Any of them.

**Comment by irco** [comment]
I have yet to find a working plunkr for md-table's. I found this post when looking for the issue:
"Can't bind to 'mdHeaderRowDef' since it isn't a known property of 'md-header-row"  in material 2.0.0-beta.8

**Comment by andrewseguin** [comment]
The fix has been made but not yet pushed out to the docs. 

https://github.com/angular/material.angular.io/pull/249

**Comment by makdeniss** [comment]
@andrewseguin any idea when it will be pushed out?

**Comment by djfishe** [comment]
This is still an issue. I get the following error in the console:

zone.min.js:1 Error: (SystemJS) Unexpected value 'undefined' exported by the module 'PlunkerMaterialModule'
	Error: Unexpected value 'undefined' exported by the module 'PlunkerMaterialModule'
	    at k (https://cdnjs.cloudflare.com/ajax/libs/zone.js/0.8.5/zone.min.js:1:12526)
	    at syntaxError (https://unpkg.com/@angular/compiler/bundles/compiler.umd.js:1729:34)
	    at eval (https://unpkg.com/@angular/compiler/bundles/compiler.umd.js:15447:40)
	    at Array.forEach (<anonymous>)
	    at CompileMetadataResolver.getNgModuleMetadata (https://unpkg.com/@angular/compiler/bundles/compiler.umd.js:15445:49)
	    at CompileMetadataResolver.getNgModuleSummary (https://unpkg.com/@angular/compiler/bundles/compiler.umd.js:15358:52)
	    at eval (https://unpkg.com/@angular/compiler/bundles/compiler.umd.js:15431:72)
	    at Array.forEach (<anonymous>)
	    at CompileMetadataResolver.getNgModuleMetadata (https://unpkg.com/@angular/compiler/bundles/compiler.umd.js:15416:49)
	    at JitCompiler._loadModules (https://unpkg.com/@angular/compiler/bundles/compiler.umd.js:26861:70)
	    at JitCompiler._compileModuleAndComponents (https://unpkg.com/@angular/compiler/bundles/compiler.umd.js:26834:36)
	    at JitCompiler.compileModuleAsync (https://unpkg.com/@angular/compiler/bundles/compiler.umd.js:26763:37)
	    at PlatformRef_._bootstrapModuleWithZone (https://unpkg.com/@angular/core/bundles/core.umd.js:4568:25)
	    at PlatformRef_.bootstrapModule (https://unpkg.com/@angular/core/bundles/core.umd.js:4554:21)
	    at execute (https://run.plnkr.co/3sdHggjfhMW8AlwO/main.ts!transpiled:113:65)
	Error loading https://run.plnkr.co/3sdHggjfhMW8AlwO/main.ts

**Comment by inthegarage** [comment]
The owners should also include cdk/stepper in the includes. I'm finding a lot of plunkers are failing as a result.

`    '@angular/cdk/stepper' : 'https://unpkg.com/@angular/cdk/bundles/cdk-stepper.umd.js',`
